public/app/plugins/datasource/loki/components/LokiExploreQueryEditor.tsx

```diff
@@ -5,29 +5,15 @@ import _ from 'lodash';
 // Types
 import { AbsoluteTimeRange, ExploreQueryFieldProps } from '@grafana/data';
 import { LokiDatasource } from '../datasource';
+import { createAbsoluteRange } from '../query_utils';
 import { LokiQuery, LokiOptions } from '../types';
 import { LokiQueryField } from './LokiQueryField';
 import LokiExploreExtraField from './LokiExploreExtraField';
 
 type Props = ExploreQueryFieldProps<LokiDatasource, LokiQuery, LokiOptions>;
 
 export function LokiExploreQueryEditor(props: Props) {
-  const { query, data, datasource, history, onChange, onRunQuery } = props;
-
-  let absolute: AbsoluteTimeRange;
```

Comment:
I have created a unified method for this and `calcAbsoluteRange` and moved it to `public/app/plugins/datasource/loki/query_utils.ts`